### DISCUSSION TITLE
How to communicate with the web service

### DISCUSSION THREAD

**Discussion by HosseinTeimouri79** [dogecoin/dogecoin#3348] [Q&A]
I am currently connecting to my dogecoin node via the command line
for example :
```
dogecoin-cli -named -rpcuser=admin -rpcpassword=1475963-test -rpcport=25555 sendtoaddress address=DMvWAyQUNJeeKr6xJpeXybVYZLi9nT2oze amount=0.1
```
or  web service
```
curl --user admin:1475963-test --data-binary '{"jsonrpc": "1.0", "id":"curltest", "method": "sendtoaddress", "params": ["DMvWAyQUNJeeKr6xJpeXybVYZLi9nT2oze ", 0.1] }' -H 'content-type: text/plain;' http://127.0.0.1:25555

```

# Output
```
{"result":null,"error":{"code":-6,"message":"Insufficient funds"},"id":"curltest"}
```
-How should I use -rpcwallet with web service(curl)?
-How should I choose the sender wallet?

**Comment by patricklodder** [comment]
Please see: https://github.com/dogecoin/dogecoin/blob/master/doc/getting-started.md#sending-transactions

It first describes the method you know - `sendtoaddress` - and subsequently walks through how to do it in a way that you can select from where to send using `listunspent`, `createrawtransaction`, `signrawtransaction` and `sendrawtransaction`
